convert2rhel/breadcrumbs.py

```diff
@@ -93,8 +93,7 @@ def _set_pkg_object(self):
 
     def _set_executed(self):
         """Set how was Convert2RHEL executed"""
-        cli_options_to_sanitize = frozenset(("--password", "-p", "--activationkey", "-k"))
```

Comment:
No need to have the options to sanitize different for logs and breadcrumbs.
Better to keep the list just on one place.